src/main/java/org/skriptlang/skript/bukkit/tags/elements/ExprTag.java

```diff
@@ -87,14 +92,21 @@ public boolean init(Expression<?>[] expressions, int matchedPattern, Kleenean is
 		nextName: for (String name : names) {
 			// get key
 			NamespacedKey key;
-			if (name.contains(":")) {
-				key = NamespacedKey.fromString(name);
-			} else {
-				// populate namespace if not provided
-				key = new NamespacedKey(namespace, name);
+			try {
+				if (name.contains(":")) {
+					key = NamespacedKey.fromString(name);
+				} else {
+					// populate namespace if not provided
+					key = new NamespacedKey(namespace, name);
+				}
+			} catch (IllegalArgumentException e) {
+				error("Invalid tag key: '" + name + "'. Tags may only contain a-z, 0-9, _, ., /, or - characters.");
+				continue;
```

Comment:
```suggestion
				key = null;
```
Could we do something like this to avoid duplicating the `error` call?